### ISSUE TITLE
new feature:  testing, jest & snapshot

### ISSUE THREAD

**Issue by diegolaciar** [janaagaard75/expo-and-typescript#23]
HI @janaagaard75,

Can you provide an example how to config jest with typescript & expo.

Thanks!

**Comment by janaagaard75** [comment]
Hi again. What have you tried so far?

**Comment by diegolaciar** [comment]
Hi @janaagaard75 ,

Yes I did. tried a lot but to be honest I don' t know to much about CRNA and new expo-cli fusion internals and the added feature Typescript, and as is lack of documentation was very hard to fugure out, also I'm newbee with RN and Jest.

I made modifications on the project based on this repo.

https://github.com/akveo/akveo-react-native-boilerplate/tree/typescript

Finally Worked!

The thing is that I don' t know if what I did is the way Expo-Cli CRNA should be configured to run test with Jest.

Thanks
